test/cql-pytest/test_json.py

```diff
@@ -58,15 +58,15 @@ def test_fromjson_wrong_type_unprepared(cql, table1):
     with pytest.raises(FunctionFailure):
         cql.execute(f"INSERT INTO {table1} (p, v) VALUES ({p}, fromJson('\"dog\"'))")
     with pytest.raises(FunctionFailure):
-        cql.execute(f"INSERT INTO {table1} (p, a) VALUES ({p}, fromJson('3'))")
+        cql.execute(f"INSERT INTO {table1} (p, a) VALUES ({p}, fromJson('[3]'))")
```

Comment:
Again, you should be proudly keeping the test which demonstrates a difference between Scylla and Cassandra (after splitting it from the part that is the same) - not replacing it with some other random test that doesn't fail... (in the previous case you changed it to an object, here a list).